addons/airway/stringtable.xml

```diff
@@ -33,6 +33,10 @@
             <Japanese>気道閉塞の確率</Japanese>
             <Portuguese>Probabilidade de obstrução das vias aéreas</Portuguese>
         </Key>
+        <Key ID="STR_KAT_Airway_SETTING_obstruction_DESC">
+            <English>Probability for an obstructed airway in persentage that gets rolled every:\n- Falling unconscious\n- Removing KingLT / Guedel Tube\n- Canceling Recovery Position</English>
```

Comment:
1) percentage
2)since when obstruction is rolled every time someone cancel recovery position ?

Reply:
yea sry my bad, changing it asap.

Reply:
can you re-add it? 😅 i forgot you added that puke func in cancel action